code/game/machinery/medical_kiosk.dm

```diff
@@ -299,7 +299,7 @@
 		brain_status = "Mild brain damage detected."  //You may have a miiiild case of severe brain damage.
 
 	if(rad_sickness_value >= 1000)  //
-		rad_sickness_status = "Patient is suffering from extreme radiation poisoning, high toxen damage expected. Suggested treatment: Repeated dosages of Pentetic Acid or high amounts of Cold Seiver and anti-toxen"
+		rad_sickness_status = "Patient is suffering from extreme radiation poisoning, high toxen damage expected. Suggested treatment: Repeated dosages of High amounts of Cold Seiver and anti-toxin"
```

Comment:
```suggestion
		rad_sickness_status = "Patient is suffering from extreme radiation poisoning, high toxin damage expected. Suggested treatment: Repeated dosages of high amounts of cold seiver and anti-toxin"
```
er, do we capitalize our chem names normally?

Reply:
I saw this, my guess is that it's to emphasize it, but at that point it should just be `<b>`.